app/src/main/res/mipmap-anydpi-v26/ic_launcher_donor_benefit.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@color/ic_launcher_donor_benefit_background"/>
```

Comment:
The `background` here can be the same as the default launcher, and no need for a custom `color` attribute.

Reply:
done.